todoie/Bindable.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Bindable.swift
+//  todoie
+//
+//  Created by Mustafa Khalil on 1/2/19.
+//  Copyright © 2019 Aurora. All rights reserved.
+//
+
+import Foundation
+
+class Bindable<T>{
+    
+    // oberver is the function that will be called when the value changes with the value
+    fileprivate var observer: ((T?)->())?
+    
+    // value is the new value inputed to the bindable object
```

Comment:
This is a normal value, with a did set function. so when the value is changed the observer is invoked.